packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -644,6 +644,22 @@
             "name": "aws_hideRegion",
             "description": "Records a call to remove a region from the explorer"
         },
+        {
+            "name": "serverlessland_createProject",
```

Comment:
why are you adding a new namespace `serverlessland_` ? why can't this use the existing `sam_` or `lambda_` namespace? using shared namespaces helps intuition and discovery.

Also, these metrics should be added to https://github.com/aws/aws-toolkit-common

Reply:
I'll go with the `lambda` as this project is not specific to `sam`

Reply:
I have opened PR ([link](https://github.com/aws/aws-toolkit-common/pull/996)) to add the telemetry to `aws-toolkit-common`